package.json

```diff
@@ -28,6 +28,7 @@
     "merge": "^1.2.0",
     "serve-static": "^1.11.1",
     "shelljs": "^0.5.0",
+    "ethereumjs-testrpc": "^3.0.3",
```

Comment:
ethereumjs-testrpc is snot included on purpose because ethereumjs has some nasty dependencies (it also looks like it's the reasons the tests in the PR are failing). We used to get a lot of issues with ppl doing the installation and they were all because of the testrpc. So it should be installed separately so people know any error on installation it's specifically due to this package and can search for a solution accordingly.
In addition for older node versions bellow 6.9, testrpc 3 will not work, and it has to be testrpc 2. which is not possible to specify in the package.json